contributors.md

```diff
@@ -106,6 +106,7 @@ The following people are not part of the project team, but have been contributin
 * Justin Gottula (jgottula)
 * Seongsik Park (pss9205)
 * (Deurklink)
+* Nathan Zabriskie
```

Comment:
Please include your username inside parentheses too, even though it's almost the same, for consistency.